pkg/util/tls/tls.go

```diff
@@ -132,7 +132,15 @@ func SetupTLSWithCertManager(caManager ClientCAManager, certManager certificate.
 	return tlsConfig
 }
 
+func SetupTLSForVirtSynchronizationControllerServer(caManager ClientCAManager, certManager certificate.Manager, externallyManaged bool, clusterConfig *virtconfig.ClusterConfig) *tls.Config {
```

Comment:
I think this needs to get a stop channel of some sort

Reply:
Do you mean the servercertmanager needs to be closed when the stop channel receives a stop? This function just returns the TLSConfig struct, doesn't do anything interesting otherwise.